### ISSUE TITLE
Number of conformer sites not matching expected XCA output

### ISSUE THREAD

**Issue by Waztom** [m2ms/fragalysis-frontend#1318]
@kaliif looking at the 2A [upload](https://fragalysis.xchem.diamond.ac.uk/viewer/react/preview/target/A71EV2A/tas/lb18145-1) we did today, @ConorFWild noticed that there should be 9 conformer sites, with only 2 recorded in the [API ](https://fragalysis.xchem.diamond.ac.uk/api/siteobservation_tag/) and two displayed in Fragalysis. 

![image](https://github.com/m2ms/fragalysis-frontend/assets/24798762/ce1ead0c-f4f1-4346-a8cb-93e9e58bb4df)

Please see conformer_sites.txt attached (equivalent of .yaml to allow uploading to an issue), product of XCA run completed for the upload. 
 
[conformer_sites.txt](https://github.com/m2ms/fragalysis-frontend/files/14112930/conformer_sites.txt)

Can you please look into this, @tdudgeon the latest tarball and upload_1 files are in Ryan's 2A (66) visit.

**Comment by Waztom** [comment]
@tdudgeon can you please check/confirm if this issue is possibly happening earlier in the compilation of one of the meta_*.yaml files?

**Comment by tdudgeon** [comment]
In data I generated today (but from data from a few days ago I see 9 conformer sites in the `meta_aligner.yaml` output:
* A71EV2A-x0152+A+201
* A71EV2A-x0309+A+147
* A71EV2A-x0379+A+147
* A71EV2A-x0395+A+147
* A71EV2A-x0488+A+147
* A71EV2A-x0517+A+147
* A71EV2A-x0525+A+147
* A71EV2A-x0853+A+147
* A71EV2A-x1081+A+147

There two canonical sites:
* A71EV2A-x0853+A+147
* A71EV2A-x1081+A+147

**Comment by kaliif** [comment]
@Waztom 9 conformer site records are in the database. The endpoint you're looking is for tags, not conformer sites directly, so it's possible there's something wrong with tag generation. What's the expected output of this API call?

**Comment by Waztom** [comment]
@kaliif there are nine conformer sites, so we expect 9 conformer sites displayed/tagged in Fragalysis. The only "tag" API endpoints I can see are:
![image](https://github.com/m2ms/fragalysis-frontend/assets/24798762/6326a1a5-2fe0-4884-8af5-bf19140a93d3)
with siteobservation_tag the only sensible endpoint I could find re-assignment of sites/tags. 

@boriskovar-m2ms can you please confirm the API endpoint you are using.

**Comment by boriskovar-m2ms** [comment]
@Waztom yes I'm using ```/api/siteobservation_tag/``` endpoint to retrieve tags from the backend

**Comment by kaliif** [comment]
@Waztom can I get a copy of the uploaded file for testing? (Tried to copy it from the stack but the file I'm getting back is  corrupt)
